bin/build-artifacts-in-docker.sh

```diff
@@ -1,37 +1,98 @@
 #!/usr/bin/env bash
 
-set -e
-set -o pipefail
-set -x
+set -eox pipefail
 
-ROOT_DIR=$(readlink -f $(dirname $0)/..)
+function usage() {
+  cat << EOF
+build-artifacts-in-docker.sh [--root_dir <dir>] [--base_image_name <name>] [--base_image_tag <tag>] [--online-dist] [--offline-dist]
+
+This script is used to build the offline and/or online distributions of presto-admin in the specified Docker image name.
+  [--root_dir <dir>]:         Directory on local filesystem where the presto-admin tree is located. Defaults to this
+                              script's current directory.
+  [--base_image_name <name>]: Name of Docker image to use during the build. Defaults to prestodev/centos6-presto-admin-tests.
+  [--base_image_tag <tag>]:   Tag of the Docker image to use during the build. Defaults to the tag stored in the base_images_tag.json file.
+  [--online-dist]:            Boolean argument indicating whether the online distribution should be build. If neither the
+                              --online-dist nor the --offline-dist flags are specified then both distributions are built.
+  [--offline-dist]:           Boolean argument indicating whether the offline distribution should be build. If neither the
+                              --online-dist nor the --offline-dist flags are specified then both distributions are built.
+EOF
+}
+
+while [[ $# -gt 0 ]]
```

Comment:
by convention, in bash scripts we use getopt, but AFAIK it supportd short options only
(for anything that is going really complex, bash isn't the tool anyway)

it's fine here to do "manual" opt parsing -- this is just test infrastructure script, so supporting `--opt arg` but not supporting `--opt=arg` is OK _here_. for production use we would need to use proper library (or proper language)